src/seedu/addressbook/AddressBook.java

```diff
@@ -794,11 +802,11 @@ private static void addPersonToAddressBook(String[] person) {
      * @return true if the given person was found and deleted in the model
      */
     private static boolean deletePersonFromAddressBook(String[] exactPerson) {
-        final boolean changed = ALL_PERSONS.remove(exactPerson);
-        if (changed) {
+        final boolean ischanged = ALL_PERSONS.remove(exactPerson);
+        if (ischanged) {
```

Comment:
the idea here is correct, naming the boolean by prefixing `is` however, you need to follow camel casing. appropriate would be `isChanged`